docker/hive/metastore-site.xml

```diff
@@ -0,0 +1,52 @@
+<configuration>
+  <property>
+      <name>metastore.thrift.uris</name>
+      <value>thrift://hive-metastore:9083</value>
+      <description>Thrift URI for the remote metastore. Used by metastore client to connect to remote metastore.</description>
+  </property>
+  <property>
+      <name>metastore.task.threads.always</name>
+      <value>org.apache.hadoop.hive.metastore.events.EventCleanerTask,org.apache.hadoop.hive.metastore.MaterializationsCacheCleanerTask</value>
+  </property>
+  <property>
+      <name>metastore.expression.proxy</name>
+      <value>org.apache.hadoop.hive.metastore.DefaultPartitionExpressionProxy</value>
+  </property>
+  <property>
+      <name>javax.jdo.option.ConnectionDriverName</name>
+      <value>com.mysql.cj.jdbc.Driver</value>
+  </property>
+  <property>
+      <name>javax.jdo.option.ConnectionURL</name>
+      <value>jdbc:mysql://localhost:3306/metastore</value>
+  </property>
+  <property>
+      <name>javax.jdo.option.ConnectionUserName</name>
+      <value>root</value>
+  </property>
+  <property>
+      <name>javax.jdo.option.ConnectionPassword</name>
+      <value>root</value>
+  </property>
+  <property>
+      <name>fs.s3a.access.key</name>
+      <value>minio</value>
+  </property>
+  <property>
+      <name>fs.s3a.secret.key</name>
+      <value>minio123</value>
+  </property>
+  <property>
+      <name>fs.s3a.endpoint</name>
+      <value>http://minio:9000</value>
+  </property>
+  <property>
+      <name>fs.s3a.path.style.access</name>
+      <value>true</value>
+  </property>
+  <property>
+      <name>hive.metastore.warehouse.dir</name>
+      <value>s3a://datalake/</value>
+  </property>
+
+</configuration>
```

Comment:
nit: new lines here and other files

Reply:
bump

Reply:
Actually there are no new lines anymore in this file.